vector/src/main/java/im/vector/riotx/features/crypto/quads/SharedSecuredStorageKeyFragment.kt

```diff
@@ -48,8 +43,7 @@ class SharedSecuredStorageKeyFragment @Inject constructor(
         ssss_restore_with_key_text.text = getString(R.string.enter_secret_storage_input_key)
 
         ssss_key_enter_edittext.editorActionEvents()
-                .debounce(300, TimeUnit.MILLISECONDS)
-                .observeOn(AndroidSchedulers.mainThread())
+                .throttleFirst(300, TimeUnit.MILLISECONDS)
```

Comment:
Why do you replace debounce by throttle? In the context of an edittext it's better to keep with debounce

Reply:
We do not observe textChange here, but editorActionEvents(), to get EditorInfo.IME_ACTION_DONE event. So I think throttleFirst is better.

Reply:
Oh yes, didn't see that